components/advancedSettings/JetpackBoost/InstallActivatePluginButton.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+import { Spinner } from "@newfold/ui-component-library";
+
+const InstallActivatePluginButton = ( { methods, constants, setModuleStatus } ) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isActive, setIsActive] = useState(false);
+  const [message, setMessage] = useState(null);
+
+  const handleInstallActivate = async () => {
+    setIsLoading(true);
+    setMessage( constants.text.jetpackBoostInstalling );
+
+    const apiUrl = methods.NewfoldRuntime.createApiUrl("/newfold-installer/v1/plugins/install");
+    const INSTALL_TOKEN = methods.NewfoldRuntime.ecommerce.install_token;
```

Comment:
We should avoid using the `ecommerce` method dependencies within the performance module. Instead, let's align it with the runtime filter approach as per one of my previous comments for better consistency and modularity.

Reply:
Commit [717557b46d8ed29411c29cd233773a943764c768](https://github.com/newfold-labs/wp-module-performance/pull/25/commits/717557b46d8ed29411c29cd233773a943764c768)